### PR TITLE
IA-2484 mount pig into Jupyter container

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-2484

TBD if this is enough or if there are more directories that need mounting. 

cc @calbach 

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/init-resources/jupyter-docker-compose.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ services:
       - /usr/lib/hadoop-mapreduce:/usr/lib/hadoop-mapreduce
       - /usr/lib/hadoop-yarn:/usr/lib/hadoop-yarn
       - /usr/lib/hive:/usr/lib/hive
-      - /usr/lig/pig:/usr/lib/pig
```
Comment (Qi77Qi) [inline http/src/main/resources/init-resources/jupyter-docker-compose.yaml]: I thikn the executable is `/usr/lib/pig/bin/pig`

```
root@saturn-29300fbd-7376-4b65-99c7-eb346f133c18-m:/usr/lib/pig/bin# pig --help
Apache Pig version 0.17.0 (r: unknown)
compiled Nov 30 2020, 11:16:03
```

Comment (rtitle) [reply http/src/main/resources/init-resources/jupyter-docker-compose.yaml]: Hm, it looks like `/usr/bin/pig` is used..
```
root@saturn-62a6b6b2-11ab-45ed-82f2-ffde0ea5da0b-m:~# which pig
/usr/bin/pig

root@saturn-62a6b6b2-11ab-45ed-82f2-ffde0ea5da0b-m:~# pig --help
Apache Pig version 0.17.0 (r: unknown) 
compiled Nov 30 2020, 11:16:03

USAGE: Pig [options] [-] : Run interactively in grunt shell.
       Pig [options] -e[xecute] cmd [cmd ...] : Run cmd(s).
       Pig [options] [-f[ile]] file : Run cmds found in file.

```

Comment (rtitle) [reply http/src/main/resources/init-resources/jupyter-docker-compose.yaml]: Will try this from the container

**Comment by rtitle** [comment]
I think this works -- I was able to do this tutorial in a Terra terminal: https://www.guru99.com/introduction-to-pig-and-hive.html

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**